src/vcpkg/commands.ci.cpp

```diff
@@ -63,8 +63,11 @@ namespace
             {
                 for (const Path& p : children)
                 {
-                    filesystem.copy_file(
-                        p, target_path / p.filename(), CopyOptions::overwrite_existing, VCPKG_LINE_INFO);
+                    const auto relative_path = StringView(p.parent_path()).substr(source_path.native().size());
```

Comment:
```suggestion
                    const auto relative_path = StringView(p).substr(source_path.native().size());
```

it seems like the substr 'dominates' any effect of parent_path?

Reply:
Can you update this to use the `proximate` things? https://github.com/microsoft/vcpkg-tool/blob/6334a4f295e8513d4b9c0a391fd761b99278cd0d/include/vcpkg/base/files.h#L240